### PR TITLE
SURF-515 Edits for footer

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

**Comment by asettle** [comment]
Waiting for images to add :)

File: _component_code/footer.md [hunk 1]
```diff
@@ -9,3 +9,57 @@ collection: component_code
...
 
 {% include todo.html content="Set up code." %}
+
+# Footer
+
+The footer lives at the bottom of the page to help visitors find
+globally applicable, but non-primary navigation information. We
+recommend using it on every page and every wrapped application. For
+customer applications, it  should not exceed the articulated scope.
+
```
Comment (asettle) [inline _component_code/footer.md]: @technabors the only thing we may need is the image for here. See the image on the overview here: https://jira.rax.io/browse/SURF-62

**Comment by araiford** [comment]
@asettle 

It looks like the footer.md file lives in the _component_code directory.  Please move the file to the _component_design directory.  I believe the _component_code dir was created before we had a plan for the code resources -- which are currently housed on the helix-ui repo here: https://rackerlabs.github.io/helix-ui/

File: _component_code/footer.md (hunk 1 above)
Comment (technabors) [reply _component_code/footer.md]: I'm really not a fan of the existing image. I'll make a new one in the same abstracted thumbnail style Ty created and upload it to the JIRA

File: _component_code/footer.md [hunk 2]
```diff
@@ -9,3 +9,57 @@ collection: component_code
 ---
 
 {% include todo.html content="Set up code." %}
+
+# Footer
+
+The footer lives at the bottom of the page to help visitors find
+globally applicable, but non-primary navigation information. We
```
Comment (robb-romans) [inline _component_code/footer.md]: instead, how about "...find global, secondary navigation information"

File: _component_code/footer.md [hunk 3]
```diff
@@ -9,3 +9,57 @@ collection: component_code
...
+recommend using it on every page and every wrapped application. For
+customer applications, it  should not exceed the articulated scope.
```
Comment (robb-romans) [inline _component_code/footer.md]: rm extra space

File: _component_code/footer.md [hunk 4]
```diff
@@ -9,3 +9,57 @@ collection: component_code
...
+
+## When to use footers
+
+Use the Rackspace footer on every application page to provide consistent
+global functionality on all of our platforms.
+
+## Best practices for footers
+
+Ensure the footer is placed at the bottom of the page, and flush to the right
```
Comment (robb-romans) [inline _component_code/footer.md]: Place the footer at the bottom of every page. 

this is unclear to me: "flush to the right of any left-hand navigation, spanning the length of the screen." 
could you rephrase?

File: _component_code/footer.md [hunk 5]
```diff
@@ -9,3 +9,57 @@ collection: component_code
...
+
+* Rackspace copyright and year
+* A link to website terms and conditions
+* A link to the privacy policy
+
+# Specifications
+
+## Footer scope
+
+Place the footer at the bottom of the navigational page.
```
Comment (robb-romans) [inline _component_code/footer.md]: The following section seems to duplicate information above. Recommend combining and rewriting.

File: _component_code/footer.md [hunk 6]
```diff
@@ -9,3 +9,57 @@ collection: component_code
...
+must contain copyright information with links to Rackspace website terms
+and privacy policy.
+
+* Website Terms: https://www.rackspace.com/information/legal/websiteterms
+* Privacy Policy: https://www.rackspace.com/information/legal/privacystatement
+
+For customer applications, do not add additional functionality to the footer.
+
+{% figure [caption:"The footer rests on the bottom of whatever page it
+  is on."] [class:"image bg-light border"] %}
```
Comment (robb-romans) [inline _component_code/footer.md]: The footer rests on the bottom of a page

File: _component_code/footer.md [hunk 7]
```diff
@@ -9,3 +9,57 @@ collection: component_code
...
+  ![]({{site.url}}/assets/images/components/content-areas/footer/footer-scope.svg){:width="80%"}
+{% endfigure %}
+
+## Footer specifications
+
+All footer content starts after the gutter at 20px. The footer container is
+40px tall with vertically centered text. Items are placed 32px between each
```
Comment (robb-romans) [inline _component_code/footer.md]: ...with text centered vertically.

Comment (robb-romans) [reply _component_code/footer.md]: Items are spaced 32px apart.

File: _component_code/footer.md [hunk 8]
```diff
@@ -9,3 +9,57 @@ collection: component_code
...
+other. Links behave according to the Links documentation, utilizing cyan 500
```
Comment (robb-romans) [inline _component_code/footer.md]: Links are colored cyan 500 on dark backgrounds and behave as described in the Links documentation.

File: _component_code/footer.md (hunk 1 above)
Comment (asettle) [reply _component_code/footer.md]: Thanks!

File: _component_code/footer.md (hunk 4 above)
Comment (asettle) [reply _component_code/footer.md]: @robb-romans - this *is* rephrased. I couldn't tell you what it meant either. @technabors halp

**Comment by asettle** [comment]
@araiford @technabors @robb-romans - updated and moved to appropriate directory with new YAML. Please review a 2nd time :)

**Comment by technabors** [comment]
Everything looks to be in the correct spot. Also I uploaded a stylized footer thumbnail to the CDN. It can be called by replacing the preview-image portion of the YAML with the below line.

```
preview-image: preview-images/footer-thumbnail.svg
```

File: _component_design/footer.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+globally applicable, but non-primary navigation information. Use it on
+every page and every wrapped application. For customer applications,
+it should not exceed the articulated scope.
+preview-image: components/preview-images/popover.svg
+resource: true
+status: in-progress
+last-modified: 2017-10-03
+---
+
+{% include todo.html content="Set up code." %}
```
Comment (nimbinatus) [inline _component_design/footer.md]: This line can be removed; it's no longer needed.

File: _component_design/footer.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
+---
+title: Footer
+parent: Navigation
+layout: component
+category: Components
+usage: The footer lives at the bottom of the page to help visitors find
```
Comment (nimbinatus) [inline _component_design/footer.md]: Add a `>` after `usage: ` and then add a soft return. It will look like this:
```yaml
usage: >
  The footer lives at the bottom of the page to help visitors find
  globally applicable, but non-primary navigation information. Use it on
  every page and every wrapped application. For customer applications,
  it should not exceed the articulated scope.
```

Alternatively, you could use a pipe (`|`). See https://stackoverflow.com/a/18708156 for details.

File: _component_design/footer.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
+---
+title: Footer
+parent: Navigation
+layout: component
+category: Components
+usage: >
+The footer lives at the bottom of the page to help visitors find
```
Comment (asettle) [inline _component_design/footer.md]: Sorry - this is a hard break, I know. Please review the rest and I will update this accordingly.

**Comment by araiford** [comment]
@asettle @technabors 

A couple things I noted when looking at the doc.  See the image below.  

Alex, if you would prefer these items be addressed in a separate PR, I'm all for that.  I hate to blow up the scope of this PR for everyone.

Lemme know, and we can merge and move on.

Drew

footer-doc

**Comment by technabors** [comment]
@araiford I vote merge this one and i'll address the preview image issue. I'll edit the source SVG that I provided Alex in the repo as well as the image from the CDN and then update the usage YAML section to reference the new image.

**Comment by nimbinatus** [comment]
Marked as approved and requested merge by @araiford.
